OpenRA.Game/World.cs

```diff
@@ -30,6 +31,8 @@ public sealed class World : IDisposable
 		internal readonly TraitDictionary TraitDict = new TraitDictionary();
 		readonly SortedDictionary<uint, Actor> actors = new SortedDictionary<uint, Actor>();
 		readonly List<IEffect> effects = new List<IEffect>();
+		readonly List<IEffect> syncedEffects = new List<IEffect>();
```

Comment:
A `List<ISync>` would make more sense - all the current loops care about the sync interface, rather than the effect interface.